modular_zubbers/code/modules/borgs/code/robot_model.dm

```diff
@@ -702,3 +746,13 @@
 #undef CYBORG_ICON_TYPE_SEC_CATBORG
 #undef CYBORG_ICON_TYPE_SYNDI_CATBORG
 #undef CYBORG_ICON_TYPE_NINJA_CATBORG
+
+//Haydeez borgs are nuts
```

Comment:
```suggestion
```

Reply:
> 

Why? This is the same as line 736 and  723. Should we not remain consistent?